app/controllers/foreman_yupana/uploads_controller.rb

```diff
@@ -8,5 +8,13 @@ def last
         output: output
       }, status: :ok
     end
+
+    def queue
+      directory = Dir["../foreman/#{ForemanYupana.base_folder}*"].select{ |f| File.file? f }.map{ |f| File.basename f }
```

Comment:
Files ready for uploads will land in [`uploads_folder`](https://github.com/laviro/foreman_yupana/blob/master/lib/foreman_yupana.rb#L9)

Reply:
Thanks, updated